### PR TITLE
For update requests alerts go to the account holder

### PR DESCRIPTION
**Pull request by amdomanska**
Fixes: https://github.com/DOAJ/doajPM/issues/2052

I am not sure how we could test it.

### CODE REVIEW

File: portality/formcontext/emails.py [hunk 1]
```diff
@@ -85,8 +85,41 @@ def send_assoc_editor_email(obj):
...
+    subject = app.config.get("SERVICE_NAME","") + " - your application has been assigned an editor for review"
+
+    alerts = []
+    for instructions in send_list:
+        to = [instructions["email"]]
+        try:
+            app_email.send_mail(to=to,
+                            fro=fro,
+                            subject=subject,
+                            template_name="email/publisher_application_editor_assigned.txt",
```
Comment (richard-jones) [inline portality/formcontext/emails.py]: I think this is the wrong template name here, you want `publisher_update_request_editor_assigned.txt`

File: portality/templates/email/publisher_application_inprogress.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Dear {{ publisher_name }},
 
-The application or update request for "{{ journal_title }}" is now being reviewed by an Associate Editor.
+The update request for "{{ journal_title }}" is now being reviewed by an Associate Editor.
```
Comment (richard-jones) [inline portality/templates/email/publisher_application_inprogress.txt]: This PR doesn't seem to have the `publisher_update_request_*.txt` templates included

Comment (amdomanska) [reply portality/templates/email/publisher_application_inprogress.txt]: That's really strange. I definitely added them here! I will fix it in a moment.

File: portality/formcontext/formcontext.py [hunk 3]
```diff
@@ -1052,7 +1060,7 @@ def finalise(self):
         # inform publisher if this was set to 'in progress' from 'pending'
         if self.source.application_status == constants.APPLICATION_STATUS_PENDING and self.target.application_status == constants.APPLICATION_STATUS_IN_PROGRESS:
             if app.config.get("ENABLE_PUBLISHER_EMAIL", False):
-                alerts = emails.send_publisher_inprogress_email(self.target)
+                alerts = emails.send_publisher_update_request_inprogress_email(self.target)
```
Comment (richard-jones) [inline portality/formcontext/formcontext.py]: I think that we need to introduce the ability to differentiate between an update request and a regular application here.  You can see this done elsewhere in this file by looking for the distinguishing mark:

```
is_update_request = self.target.current_journal is not None
```

File: portality/templates/email/publisher_update_request_editor_assigned.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Dear {{ publisher_name }},
+
+Yourupdate request for "{{ application_title }}" has been assigned to an editor for review for inclusion in the DOAJ.
```
Comment (richard-jones) [inline portality/templates/email/publisher_update_request_editor_assigned.txt]: missing space `Your update`

File: doajtest/unit/test_formcontext_emails.py [hunk 5]
```diff
@@ -278,15 +278,15 @@ def test_01_maned_review_emails(self):
                                         re.DOTALL)
         assert bool(assEd_email_matched)
 
-        publisher_template = 'publisher_application_editor_assigned.txt'
+        publisher_template = 'publisher_update_request_editor_assigned.txt'
```
Comment (richard-jones) [inline doajtest/unit/test_formcontext_emails.py]: It looks like this test no longer checks to see if `publisher_application_editor_assigned` is ever triggered now.

File: doajtest/unit/test_formcontext_emails.py [hunk 6]
```diff
@@ -464,15 +464,15 @@ def test_02_ed_review_emails(self):
...
```
Comment (richard-jones) [inline doajtest/unit/test_formcontext_emails.py]: Same as above, looks like this test no longer checks to see if `publisher_application_editor_assigned` is triggered.

**Pull request closed without merging**